src/bluehawk/parser/lexer/makeLexer.ts

```diff
@@ -38,3 +37,16 @@ export function makeLexer(languageTokens: TokenType[]): Lexer {
     defaultMode: "RootMode",
   });
 }
+
+// Helper function for generating command attribute pop tokens
+function _CommandAttributesPopTokens(languageTokens: TokenType[]): TokenType[] {
```

Comment:
Minor: Why `_`?

Reply:
Sorry forgot that in JS you need to explicitly export functions rather than private being a convention in the name of the function like in python.